### PR TITLE
code collation

### PR DESCRIPTION
**Pull request by lucming**
…les in the preemption plugin

### Ⅰ. Describe what this PR does
1. optimise the handling of error
2. optimise the initialisation logic of variables in the preemption plugin

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/scheduler/plugins/reservation/garbage_collection.go [hunk 1]
```diff
@@ -62,16 +62,15 @@ func (p *Plugin) gcReservations() {
...
-		if isReservationNeedCleanup(r) {
-			err = p.client.Reservations().Delete(context.TODO(), r.Name, metav1.DeleteOptions{})
-			if err != nil {
-				klog.V(3).InfoS("failed to delete reservation", "reservation", klog.KObj(r), "err", err)
-				continue
-			}
-
-			p.reservationCache.Delete(r)
+		if err = p.client.Reservations().Delete(context.TODO(), r.Name, metav1.DeleteOptions{}); err != nil {
+			klog.V(3).InfoS("failed to delete reservation", "reservation", klog.KObj(r), "err", err)
```
Comment (eahydra) [inline pkg/scheduler/plugins/reservation/garbage_collection.go:70]: Use `Error` instead of `InfofS`?

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**